.github/workflows/run-test.yml

```
@@ -1,9 +1,18 @@
 name: Run Mergin Plugin Tests
 
-on: [push]
+on: 
+  push:
+  workflow_dispatch:
+    inputs:
+      MERGIN_CLIENT_VER:
```

Let's not introduce old namings :) 
```suggestion
      API_CLIENT_VER:
```

---

The same below "Mergin client version either ..."

---

Renamed all the occurences I settled for `PYTHON_API_CLIENT_VER` in the end